### PR TITLE
Move documentation to Paradox (#50)

### LINKED ISSUE #50: Move documentation to Paradox
Let's co-ordinate and move the docs of ssl-config to paradox as well ( https://github.com/lightbend/paradox ) .
I'm hoping to cross link to it from akka docs (and perhaps at that point in time play could do so as well).

cc @eed3si9n @wsargent

### PR DESCRIPTION
**Pull request by jonas**
The result can be viewed here: https://jonas.github.io/ssl-config/index.html

Fixes #50

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -47,7 +47,8 @@ sbt '+publishSigned'
 
 echo "Publishing docs..."
 sbt makeSite
-mv documentation/target/sphinx/html ../releasing-the-docs
+rm -rf documentation/target/paradox/paradox # Paradox site bug?
```
Comment (jonas) [inline release.sh]: This is a workaround for what might be a bug in the sbt-site Paradox plugin. If I have time I will try to investigate later.

**Comment by ktoso** [comment]
Wohoo, awesome :)

File: release.sh (hunk 1 above)
Comment (ktoso) [inline release.sh]: Best to report on paradox repo I think. Issue is that it should have cleaned up that directory by itself right?

**Comment by ktoso** [comment]
LGTM, lookin good.
I wonder if @2m would be able to set this up to publish docs into doc-hub?

**Comment by jonas** [comment]
Did one last pass

File: documentation/src/paradox/WSQuickStart.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+known certificate authority, then WS should work out of the box without
+any additional configuration. You're done!
+
+If the web service is not using a well known certificate authority, then
+it is using either a private CA or a self-signed certificate. You can
+determine this easily by using curl:
+
+```
+curl https://financialcryptography.com # uses cacert.org as a CA
+```
```
Comment (2m) [inline documentation/src/paradox/WSQuickStart.md]: ssl

Syntax highlighting thinks that `//` is a start of a comment. Can we adjust that by giving a hint after ```?

File: documentation/src/paradox/WSQuickStart.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+ of Certificate Authority (CA) public keys (CA certs). If the default
+ bundle file isn't adequate, you can specify an alternate file
+ using the --cacert option.
+If this HTTPS server uses a certificate signed by a CA represented in
+ the bundle, the certificate verification probably failed due to a
+ problem with the certificate (it might be expired, or the name might
+ not match the domain name in the URL).
+If you'd like to turn off curl's verification of the certificate, use
+ the -k (or --insecure) option.
+```
```
Comment (2m) [inline documentation/src/paradox/WSQuickStart.md]: Here as well text after `'` is highlighted differently.

ssl_test

Comment (jonas) [inline documentation/src/paradox/WSQuickStart.md]: I've opened lightbend/paradox#64 to fix this

File: documentation/src/paradox/WSQuickStart.md (hunk 2 above)
Comment (jonas) [inline documentation/src/paradox/WSQuickStart.md]: I marked this as `sh` to fix in jonas@f6509448fc4e73d7227584d7943f11535ffd65e9

File: documentation/src/paradox/TestingSSL.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+Testing an SSL client not only involves unit and integration testing,
+but also involves adversarial testing, which tests that an attacker
+cannot break or subvert a secure connection.
+
+## Unit Testing
+
+Play comes with `play.api.test.WsTestClient`, which provides two
+methods, `wsCall` and `wsUrl`. It can be helpful to use
+`PlaySpecification` and `in new WithApplication`
```
Comment (jonas) [inline documentation/src/paradox/TestingSSL.md]: Agree, right now I don't think the scaladocs are published.

**Comment by jonas** [comment]
I don't know what limitation GitHub puts on their pages, but one idea would be to have a separate `docs-validation` repo or organization and push generated docs to its GitHub pages using the PR and/or Travis build ID as the path.

**Comment by ktoso** [comment]
What's missing here to make this happen; the release process I guess?

// @2m

**Comment by 2m** [comment]
I think we can merge this one.

The release process can be adjusted in a separate PR.

**Comment by ktoso** [comment]
Boom, thanks

**Comment by ktoso** [comment]
Kudos guys for migrating this!

**Pull request merged**